src/pkg/reg/adapter/harbor/base/adapter.go

```diff
@@ -224,6 +224,10 @@ func (a *Adapter) ListProjects(filters []*model.Filter) ([]*Project, error) {
 		names, ok := util.IsSpecificPathComponent(projectPattern)
 		if ok {
 			for _, name := range names {
+				if strings.Contains(name, " ") {
```

Comment:
or can we use strings.TrimSpace() to ensure name? And let user to config white space in UI? Just a suggestion.